docs/user/openstack/customization.md

```diff
@@ -137,6 +138,12 @@ platform:
       clusterOSImage: my-rhcos
 ```
 
+## Custom Subnets
+
+Before running the installer with a custom `machinesSubnet`, there are a few things you should check. First, make sure that you have a network and subnet available. Note that the installer will create ports on this network, and some ports will be given a fixed IP addresses in the subnet's CIDR. Make sure that the credentials that you give the installer are authorized to do this! The installer will also apply a tage to the subnet. Lastly, the installer will not create a router, so make sure to do this if you need a router to attach floating ip addresses to the network you want to install the nodes onto. Note that for now, this feature does not fully support provider networks. 
```

Comment:
s/tage/tag/
Do we track the support (or lack of) custom subnets and provider networks somewhere? Not asking to put the link in the doc.

Reply:
I am not sure, but this has come up a few times as a common misunderstanding. If not here, we should make a note of it somewhere at least